src/Mvc/src/Microsoft.AspNetCore.Mvc.ViewFeatures/Microsoft.AspNetCore.Mvc.ViewFeatures.csproj

```diff
@@ -21,6 +21,7 @@ Microsoft.AspNetCore.Mvc.ViewComponent</Description>
     <Reference Include="Microsoft.AspNetCore.Diagnostics.Abstractions" />
     <Reference Include="Microsoft.AspNetCore.Html.Abstractions" />
     <Reference Include="Microsoft.Extensions.WebEncoders" />
+    <PackageReference Include="Microsoft.AspNetCore.Components" Version="3.0.0-preview-t000" />
```

Comment:
This will be replaced by `<Reference Include="Microsoft.AspNetCore.Components" />`

Reply:
Done